app/workers/education_form/process10203_submissions.rb

```diff
@@ -64,7 +64,10 @@ def process_user_submissions(user_submissions)
         log_info "EDIPI available for process STEM claim ids=#{claim_ids}: #{auth_headers&.key?('va_eauth_dodedipnid')}"
 
         gi_bill_status = get_gi_bill_status(auth_headers)
-        poa = get_user_poa_status(auth_headers)
+        poa = submissions.last.education_stem_automated_decision.poa
+
+        # only check EVSS if poa wasn't set on submit
+        poa = get_user_poa_status(auth_headers) if poa.nil?
```

Comment:
you could get rid of the `if` statement here and clean it up a bit by doing:
`poa = submissions.last.education_stem_automated_decision.poa || get_user_poa_status(auth_headers)`